nxdrive/data/i18n/i18n.json

```
@@ -88,6 +88,7 @@
     "DIRECT_TRANSFER_WINDOW_TITLE": "Direct Transfer - %1",
     "DISCONNECT": "Remove account",
     "DISCONNECTING": "Disconnecting …",
+    "DRIVE_DISK_SPACE_TOOLTIP": "Drive space: %1",
```

```suggestion
    "DRIVE_DISK_SPACE_TOOLTIP": "%1 space: %2",
```
Where `%1` will be the `AAP_NAME`.